pkgs/tools/package-management/cargo-edit/default.nix

```diff
@@ -15,15 +14,25 @@ rustPlatform.buildRustPackage rec {
 
   cargoSha256 = "1h1sy54p7zxijydnhzvkxli90d72biv1inni17licb0vb9dihmnf";
 
-  nativeBuildInputs = [ pkg-config ];
-  buildInputs = [ openssl ] ++ lib.optionals stdenv.isDarwin [ libiconv darwin.apple_sdk.frameworks.Security ];
+  nativeBuildInputs = [
+    pkg-config
+    xcbuild # the cc crate attempts to run xcbuild
```

Comment:
This should not be necessary once staging is merged in master:

https://github.com/NixOS/nixpkgs/pull/97000

Maybe add a reference + comment that this should be removed once that PR lands?

Also, the `xcbuild` dependency should (probably) be conditional so that's only used on Darwin.

Reply:
pushed a fix, let me know if that works for you.

Reply:
Thanks! I have pushed a change that makes the formatting a little more canonical. While at it, I tacked on another commit that passes the `Security` framework in the preferred manner.

Reply:
Thanks!
I didn't know that was the prefered manner. Will keep it in mind for next time.